server/batch_controller_test.go

```diff
@@ -35,24 +33,29 @@ func (s *BatchControllerSuite) SetUpSuite(c *C) {
 
 	// Set up the database
 	var err error
-	s.Session, err = mgo.Dial("localhost")
-	util.CheckErr(err)
-	s.Database = s.Session.DB("fhir-test")
+	s.Connection = new(MongoConnection)
+	err = s.Connection.Connect("localhost")
+
+	if err != nil {
+		panic(err)
+	}
```

Comment:
This is what `util.CheckErr(err)` is for (see deleted code in the diff). It's cleaner.